Loops/loops_homework_6.py

```diff
@@ -0,0 +1,49 @@
+list_some_elements = [23, 0, 56, 100, 50, True, False, 'abc', 10.5]
+client_num = int(input("Please, enter your number: "))
+dividing_list_num = []
+for elements in list_some_elements:
+    if type(elements) == int:
+        dividing_num = elements // client_num
+        dividing_list_num.append(dividing_num)
+
```

Comment:
I am not sure that your decision is valid
Should be like this
![image](https://github.com/Pticatos86/test_repos_python/assets/51053483/b8e5cc20-75b7-4837-9ae8-8c86877e3ad8)

Reply:
fixed